### PR TITLE
Migrated Storybook notation from CSF2 to CSF3

### PR DESCRIPTION
**Pull request by kamegoro**
## Summary

Migration of Stories files in Generic and Presentational directories to CSF3 notation

issue: https://github.com/NiceNode/nice-node/issues/404

### CODE REVIEW

File: src/stories/Generic/DynamicSettings.stories.tsx [hunk 1]
```diff
@@ -8,13 +8,7 @@ import DynamicSettings, {
 export default {
   title: 'Generic/DynamicSettings',
   component: DynamicSettings,
-  argTypes: {},
-} as ComponentMeta<typeof DynamicSettings>;
-
-const Template: ComponentStory<typeof DynamicSettings> = (args) => (
-  <DynamicSettings {...args} />
-);
-console.log('yoooooo');
```
Comment (kamegoro) [inline src/stories/Generic/DynamicSettings.stories.tsx:17]: Deleted because debug logs remained.

File: src/stories/Generic/Modal.stories.tsx [hunk 2]
```diff
@@ -1,18 +1,10 @@
...
 
-const Template: ComponentStory<typeof Modal> = (args) => <Modal {...args} />;
-
-export const Primary = Template.bind({});
-// Primary.args = {
-//   screen: {
-//     route: 'addNode',
-//     type: 'modal',
-//   },
-// };
```
Comment (kamegoro) [inline src/stories/Generic/Modal.stories.tsx:18]: Commented out code, so I removed it once.

**Review by kamegoro — COMMENTED** [review]
I used Storybook's migration command.
I have also checked that it works in preview mode, but I have commented the code where I made changes of my own volition.

**Comment by jgresham** [comment]
Thanks for taking the initiative and getting this done!

**Pull request merged**